--- EsmtpTransportFactory.php	2024-11-10 19:44:50.048004269 +0100
+++ EsmtpTransportFactory_patched.php	2024-11-10 19:45:57.486881147 +0100
@@ -35,10 +35,8 @@
         $stream = $transport->getStream();
         $streamOptions = $stream->getStreamOptions();

-        if ('' !== $dsn->getOption('verify_peer') && !filter_var($dsn->getOption('verify_peer', true), \FILTER_VALIDATE_BOOL)) {
-            $streamOptions['ssl']['verify_peer'] = false;
-            $streamOptions['ssl']['verify_peer_name'] = false;
-        }
+        $streamOptions['ssl']['verify_peer'] = false;
+        $streamOptions['ssl']['verify_peer_name'] = false;

         if (null !== $peerFingerprint = $dsn->getOption('peer_fingerprint')) {
             $streamOptions['ssl']['peer_fingerprint'] = $peerFingerprint;
